### ISSUE TITLE
Can't challenge a proposal with just GIV on the giveth garden - need HNYT?

### ISSUE THREAD

**Issue by laurenluz** [1Hive/gardens#138]
Can't seem to challenge a proposal with just GIV on the Giveth Garden, requires HNYT to dispute the proposal... shouldn't I be able to challenge without HNYT and then the proposal owner must put up HNYT to dispute?

also... where can I get HNYT?

![image.png](https://images.zenhubusercontent.com/5fdfac16a58523a866ddb68d/fa92e510-a192-490a-93e9-470dfe5958cf)

**Comment by GriffGreen** [comment]
Yeah.. i thought the first challenge token was done in the Gardens token, and then HNY was used for Celeste?

**Comment by fabriziovigevani** [comment]
The way the covenant works is such that the losing party of the dispute ends paying the dispute fees so both challenger and submitter need to deposit the dispute fees. After the dispute is resolved, the winning party will get the dispute fess refund.

We don't have a faucet currently for the HNYT, what's your address? I'll send you some.

**Comment by GriffGreen** [comment]
@fabriziovigevani 0x839395e20bbB182fa440d08F850E6c7A8f6F0780

**Comment by fabriziovigevani** [comment]
Thanks! Just sent you some.

**Comment by 0xGabi** [comment]
Here you can find the designs to address this issue:
https://www.figma.com/file/N0JPKfIM0LlYaJInLJt3Eq/Swap-HNY-from-Challenge-modal?node-id=2%3A1967
